core/vm/ipgraph.go

```diff
@@ -29,11 +36,41 @@ var (
 type ipGraph struct{}
 
 func (c *ipGraph) RequiredGas(input []byte) uint64 {
-	return uint64(1)
```

Comment:
I think we should use block height to guard the changes in the ipgraph.go file. Before the block height, using the previous version. After the block height, using the new version.

Reply:
As discussed offline and in reference to the Geth PR regarding the gas cost changes, it would be still safe to not guarding the gas cost adjustment based on block height.